lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +20,13 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    text_punctuation = text.translate(str.maketrans('', '', string.punctuation))
+    text_strip = text_punctuation.strip()
```

Comment:
is it necessary to strip if later we split by whitespaces?

Reply:
done